expect/expect.ts

```diff
@@ -94,7 +98,7 @@ const matchers: Record<MatcherKey, Matcher> = {
   toThrow,
 };
 
-export function expect(value: unknown, customMessage?: string): Expected {
+export const expect = (value: unknown, customMessage?: string): Expected => {
```

Comment:
We have a style guide about this. We prefer `function` over arrow syntax https://docs.deno.com/runtime/manual/references/contributing/style_guide#top-level-functions-should-not-use-arrow-syntax